CordovaLib/CordovaLib.docc/upgrading-8.md

```diff
@@ -125,18 +125,15 @@ You can still access the `scrollView` property of the web view by dynamically in
 
 ```objc
 // Old code
-UIScrollView *scroller = self.webView.scrollView;
+UIScrollView *scrollView = self.webView.scrollView;
 ```
 
 ```objc
 // New code (Objective-C)
 #import <objc/message.h>
```

Comment:
We probably don't need this import anymore either, since we're not calling `objc_msgSend`

Reply:
Ah ok understand, I remove this